vmdb/app/models/miq_user_role.rb

```diff
@@ -133,10 +133,14 @@ def self.allows_all_children?(role, options={})
   end
 
   def self.get_role(role)
-    return role if role.kind_of?(self) || role.nil?
-
-    find_method = role.kind_of?(Integer) ? :find_by_id : :find_by_name
-    self.send(find_method, role, :include => :miq_product_features)
+    case role
+    when self, nil
+      role
+    when Integer
+      includes(:miq_product_features).find_by_id(role)
+    else
+      includes(:miq_product_features).find_by_name(role)
```

Comment:
There was certainly at least one, to lead me to line 139's `:include`. I can make no assertion on how comprehensively covered the method is overall... the `send` just made me sad.